examples/ownership-transfer/src/index.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
```

Comment:
Will the node interpreter automatically run typescript without any modification when a ts script with this shebang line is invoked?

Reply:
Good catch @zonotope - leftover from before I rewrote to TS.